package.json

```diff
@@ -2,10 +2,9 @@
   "dependencies": {},
   "devDependencies": {
     "assert-no-diff": "3.0.3",
-    "harp": "0.30.0",
+    "harp": "0.31.0",
```

Comment:
Is this used for building the website? Any verification needed for this?

Reply:
Yes, Harp is renders our website. At some point, we might want to switch to https://docusaurus.io since Harp is no longer maintained and the website looks dated and isn't that useful.